docs/user-guide/compute-resources.md

```diff
@@ -62,23 +65,28 @@ distinct from [API resources](working-with-resources.md).  API resources, such a
 [services](services.md) are objects that can be written to and retrieved from the Kubernetes API
 server.
 
-## Container and Pod Resource Limits
+## Resource Requests and Limits of Pod and Container
 
 Each container of a Pod can optionally specify `spec.container[].resources.limits.cpu` and/or
-`spec.container[].resources.limits.memory`.  The `spec.container[].resources.requests` field is not
-currently used and need not be set.
-
-Specifying resource limits is optional.  In some clusters, an unset value may be replaced with a
-default value when a pod is created or updated.  The default value depends on how the cluster is
-configured.
-
-Although limits can only be specified on individual containers, it is convenient to talk about pod
-resource limits.  A *pod resource limit* for a particular resource type is the sum of the resource
-limits of that type for each container in the pod, with unset values treated as zero.
-
-The following pod has two containers.  Each has a limit of 0.5 core of cpu and 128MiB
-(2<sup>20</sup> bytes) of memory.  The pod can be said to have a limit of 1 core and 256MiB of
-memory.
+`spec.container[].resources.limits.memory`.  The `spec.container[].resources.requests` field can
```

Comment:
I would just rewrite this paragraph to 

Each container of a Pod can optionally specify \`spec.container[].resources.limits.cpu\` and/or
\`spec.container[].resources.limits.memory\` and/or  \`spec.container[].resources.requests.cpu\`
and/or  \`spec.container[].resources.requests.memory\`.